history-of-quakers/modernized/13-chapter-11.adoc

```diff
@@ -3078,8 +3183,16 @@ We do beseech you to tell all our dear friends, fathers, and elders,
 the pillars of our spiritual building, with all the rest of our Christian brethren,
 that we do desire their prayers, for we have need of them.
 
+--
+
+[.offset]
 This letter they concluded with the following Poem:
 
+[.embedded-content-document.letter]
+--
+
+[verse]
+____
```

Comment:
This poem -- can you find it in the original and give it a quick once over? I noticed from looking at the PDF that it's got some missing spaces, and I wondered if some of the strange capital letters meant that the lines should be broken more often.  Maybe see if you can get the lines the same as the original and fix anything you notice, like `weep:Our` and `bright,Who`, etc. Thanks!